pkg/print/common/errors.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+import "fmt"
+
+// ResultError is returned from printers when the apply/destroy operations completed, but one or
+// more resources either failed apply/prune/delete, or failed to reconcile.
+type ResultError struct {
+	ApplyFailed  int
+	PruneFailed  int
+	DeleteFailed int
+
+	ReconcileTimeout int
+	ReconcileFailed  int
+}
+
+func (a *ResultError) Error() string {
+	failedActuateSum := a.ApplyFailed + a.PruneFailed + a.DeleteFailed
+	failedReconcileSum := a.ReconcileTimeout + a.ReconcileFailed
+	switch {
+	case failedActuateSum > 0 && failedReconcileSum > 0:
+		return fmt.Sprintf("%d resources failed, %d resources failed to reconcile before timeout",
```

Comment:
`%d resources failed, %d resources failed to reconcile before timeout` -> `%d resources failed to actuate, %d resources failed to reconcile before timeout`

Reply:
This would be an output change to kpt. Without this, the change is reverse compatible.

I'm not sure we want to expose the "actuate" or "actuation" terms to users yet. We'll want to be very deliberate about it and what it means if/when we do.

Reply:
I also don't think we should change the language here.